ansible_deployer/command_line.py

```diff
@@ -679,12 +682,14 @@ def get_all_user_groups():
 
     return user_groups
 
-def load_global_configuration():
+def load_global_configuration(options: dict):
     """Function responsible for single file loading and validation"""
     check_cfg_permissions_and_owner(APP_CONF)
     with open(APP_CONF, "r", encoding="utf8") as config_stream:
         try:
             config = yaml.safe_load(config_stream)
+            if options["config"]:
```

Comment:
move it to the beginig of the function - if we open option speciied config we should not verify APP_CONF permissions or attempt to open it. It may even not exist in this case.